contributing/code_of_conduct.md

```diff
@@ -0,0 +1,89 @@
+_This code-of-conduct is common to the following projects:_
+* _[Pagerfanta](https://github.com/whiteoctober/Pagerfanta)_
+* _[Pagerfanta Bundle](https://github.com/whiteoctober/WhiteOctoberPagerfantaBundle)_
+* _[SwiftMailerDBBundle](https://github.com/whiteoctober/WhiteOctoberSwiftMailerDBBundle)_
+* _[BreadcrumbsBundle](https://github.com/whiteoctober/breadcrumbsbundle)_
+* _[TCPDFBundle](https://github.com/whiteoctober/WhiteOctoberTCPDFBundle)_
```

Comment:
what's that?

Reply:
a code of conduct. pretty standard for open source projects. [gitbook](https://www.gitbook.com/pricing) ask the projects to have one for community plan